### PR TITLE
Update templates dependencies

### PR DESCRIPTION
**Pull request by pontusva**

### CODE REVIEW

File: package-lock.json
Comment (szymmis) [inline package-lock.json:1]: `vite-express` is using `yarn` as a package manager so you need to remove this file, and make sure that `yarn install` is run to update `yarn.lock` file correctly.

**Review by szymmis — COMMENTED** [review]
Great job! I've tested it locally and it looks like every dependency is updated, thanks a lot.

As I stated in a comment bellow, we need to remove `package-lock.json` as `yarn` is used here as a package manager.

Other tip: I saw that you've changed the PR name and indeed that was good decision, but the title isn't quite good enough yet. Always remember that the best commit/pr title states what changed, so that reader can easily understand what was changed. Much better title would be something like: `"Update templates dependencies"` 

When it comes to the deprecation message: I'll try to make an update after this PR is merged.

File: package-lock.json
Comment (pontusva) [reply package-lock.json:1]: package-lock.json is removed now.

**Comment by pontusva** [comment]
Def need to check this

**Comment by szymmis** [comment]
Yeah, it was failing because `Vite 5` needs node 18 or 20, and we were using 16 in the CI. I've created and merged bump in #95, so you can now try to pull the changes and rebase. Could you do that for me?

In case you don't know how to do this:
1. Checkout to `master` branch in your project
2. `git fetch`
3. `git pull`
4. Checkout to the `update-dependencies` branch
5. `git rebase master`
6. `git push --force-with-lease`

**Comment by pontusva** [comment]
> Yeah, it was failing because `Vite 5` needs node 18 or 20, and we were using 16 in the CI. I've created and merged bump in #95, so you can now try to pull the changes and rebase. Could you do that for me?
> 
> In case you don't know how to do this:
> 
>     1. Checkout to `master` branch in your project
> 
>     2. `git fetch`
> 
>     3. `git pull`
> 
>     4. Checkout to the `update-dependencies` branch
> 
>     5. `git rebase master`
> 
>     6. `git push --force-with-lease`

I already merged master into main without rebasing. Should i revert that merge and then rebase instead?

**Comment by szymmis** [comment]
No you don't have, it should be fine with merge also. With merge commit you can just `git push` then I guess.

**Comment by pontusva** [comment]
thanks for letting me know about rebase. Will look into that

**Review by szymmis — APPROVED** [review]
Looks like everything is working fine, great job! :tada: 
Thanks a lot for your contribution!

> thanks for letting me know about rebase. Will look into that

To be honest it's just a preference thing, you can look into a `rebase vs merge` debate, but it the end it comes down to the project and people you are working on.

**Comment by szymmis** [comment]
Also, your changes won't be available immediately because there is still work around getting rid off the warning to do, so I'll let you know here when the update will be ready.

**Comment by pontusva** [comment]
I could try see if i can fix the warning as well. It would be a fun challange :) I'll look into it later today. Should open another issue for this?

**Comment by szymmis** [comment]
I think that you can just open a PR when you have changes, but if you want some guidance how to work on those warnings, then you can open an issue for clarity. I've did some research yesterday so I have a little bit of knowledge about how to handle that, so I'll be happy to share it with you.

**Comment by pontusva** [comment]
yes I think some guidance would be helpful :) Thanks!

**Pull request merged**